src/dune_rules/clflags.mli

```diff
@@ -19,7 +19,7 @@ val debug_artifact_substitution : bool ref
 (** Print package output when building with package management *)
 val debug_package_logs : bool ref
 
-(** Whether we are ignoring "dune.lock/". *)
+(** Whether we are ignoring ".dune-solution-cache/". *)
```

Comment:
Maybe update this comment to not mention the value. Especially since it doesn't just ignore `dune.lock`, it ignores every lock directory.